assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -30,6 +32,11 @@ export default Ember.Controller.extend({
   sortBy: ["last_run_at:desc"],
   sortedQueries: Ember.computed.sort("model", "sortBy"),
 
+  @computed
+  acceptedImportFileTypes() {
+    return ["application/json"];
```

Comment:
Thanks for changing this. I think the other modal was done in 2014 so maybe this wasn't the accepted mime type back then? Anyway application/json is totally fine, never used anything but that.